libraries/AP_HAL_Linux/SPIDevice.cpp

```diff
@@ -112,6 +112,13 @@ SPIDesc SPIDeviceManager::_device[] = {
     SPIDesc("lsm9ds0_g",  0, 0, SPI_MODE_3, 8, RPI_GPIO_12,  10*MHZ, 10*MHZ),
     SPIDesc("raspio",     0, 0, SPI_MODE_3, 8, RPI_GPIO_7,   10*MHZ, 10*MHZ),
 };
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ALTERA
+SPIDesc SPIDeviceManager::_device[] = {
+    /* MPU9250 is restricted to 1MHz for non-data and interrupt registers */
+	SPIDesc("mpu9250",    32765, 1,  SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 1*MHZ),
+	SPIDesc("ms5611",     32766, 0,  SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 1*MHZ),
+	SPIDesc("ublox",      32766, 1,  SPI_MODE_0, 8, SPI_CS_KERNEL,  5*MHZ, 5*MHZ),
```

Comment:
32765?? Is this correct? I've never seen a spi bus with such a high number?  Could you change your udev/mdev to rename the device to a lower number? The reason I'm asking this is that we are starting to identify sensor changes from one bus to another. We have only a few bytes to hold the bus number + CS.... 32765 will definitely not fit.